### ISSUE TITLE
Enhancement: Prevent endercrystal pvp damage

### ISSUE THREAD

**Issue by stevmei** [Kakifrucht/WorldGuardFix#15]
Hey!

It it possible to fix player getting damage when a endercrystal is destroyed by another player in a `pvp: deny` area?

Since WG b1682 endercrystal block damage is blocked with the `block-other-explosions: true`, but it is still possible to kill players by destroying a nearby endercrystal.

That would be very nice :smile:

**Comment by Kakifrucht** [comment]
The purpose of WorldGuardFix is/was to fix the lacking features in WorldGuard before the official updates were released. This seems more like a feature request than a fix and is thus not really compatible with what this tries to achieve. Additionally as long as SpigotMC's plugin review system is in such a broken state I don't really plan to keep supporting the plugin.

Thanks for letting me know about the endercrystal fix. For now the only remaining use for WorldGuardFix is to fix the buggy potion block list. I guess I'll update if a new Minecraft/Spigot Update comes out that needs new protections and @wizjany decides not to update WorldGuard in a timely manner. Until then I don't plan to add additional features if not really necessary.

Thanks for the suggestion.

**Comment by wizjany** [comment]
Has been an actively tracked issue for a while for us, although was not fixed in a "timely manner" due to hazy communication. http://dev.enginehub.org/youtrack/issue/WORLDGUARD-2620

Would help if people who actually run servers and experience issues file them through the correct means instead of completely subverting the official plugin. We won't know there's bugs unless someone tells us.

**Comment by stevmei** [comment]
Thanks again @wizjany for fixing this in the official build. This issue was just created for the opportunity that you'll take a longer time to fix this. But you fixed it today and that's nice :smile:

@Kakifrucht: I dunno if the potion block list is already fixed in the official build since I'm not using this feature :smile:, so for me there's no reason to use WGF anymore.

But WGF was a very good workaround for many annoying bugs and it was the one and only reason why my server was updated to 1.9 very early. Thanks again for this plugin. From a german developer to a german developer.

I guess, I'm gonna tell the owner of my server that he has to make a little donation to you. I've seen the 'Buy me a coffee' link :smile:

**Comment by Kakifrucht** [comment]
@wizjany This plugin was created when there was no set ETA for WorldGuard 1.9 support, the issues I fixed were already reported at the time. What I meant was that you guys didn't want to set an ETA and if that happens again, I will keep updating this plugin to fix the reported issues. You fixed the issues (beside the buggy potion blocklist) all as far as I can tell. It was never meant to subvert the official plugin, I just decided to not pull request on WorldGuard because it would have been significantly more work for me, as I am/was not experienced with WorldGuard's code base.

@sigmaroot thanks but not necessary ;)
Your donation might be in better hands if you channel it through WorldGuard's donation pages, @wizjany did significantly more than I did with this small fix tool.
